substratevm/src/com.oracle.svm.configure.test/src/com/oracle/svm/configure/test/config/ResourceConfigurationTest.java

```diff
@@ -0,0 +1,98 @@
+package com.oracle.svm.configure.test.config;
```

Comment:
I ran your PR in our CI. 

It complains that this file is missing the license header.